lib/routes/create_invoice/create_invoice_page.dart

```diff
@@ -54,6 +55,7 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
   final _amountController = TextEditingController();
   final _amountFocusNode = FocusNode();
   var _doneAction = KeyboardDoneAction();
+  bool channelCreationPossible = false;
```

Comment:
It is better not to save this flag as part of the state.  Since we are using the lspbloc builder in the build method we can pass the flag directly from there.
We will also get the advantage of not having to make it optional.

Reply:
Good point. Done